### DISCUSSION TITLE
drag and drop problem

### DISCUSSION THREAD

**Discussion by ElyesBenCheikh1** [xyflow/xyflow#2122] [Q&A]
Hi 
i am working with reactflow and i have some issues. I have a problem with the drag and drop example.
this is the error: TypeError: Cannot read properties of null (reading 'getBoundingClientRect').
I can create a codesandbox if it will make it easier to understand me.

Thank you.

**Comment by moklick** [comment]
Hey! A codesandbox would be very helpful, thanks!
